cmd/legacy/legacyparams/params.go

```diff
@@ -133,6 +145,20 @@ func loadAPIParams(v *viper.Viper) (API, error) {
 	apiURL = strings.TrimSuffix(apiURL, "/heartbeats")
 	apiURL = strings.TrimSuffix(apiURL, "/heartbeat")
 
+	var backoffAt *time.Time
+
+	backoffAtStr := vipertools.GetString(v, "internal.backoff_at")
+	if backoffAtStr != "" {
+		parsed, err := time.Parse(config.DateFormat, backoffAtStr)
+		if err != nil {
+			log.Warnf("failed to parse backoff at to time instance: %s", err)
+		}
+
+		backoffAt = &parsed
```

Comment:
I guess in case of failing to parse the string, it is better to ignore the returned value. Although, in go if/else is mostly avoided, I don't think, we can get around it here. 
```suggestion
		if err != nil {
			log.Warnf("failed to parse backoff at to time instance: %s", err)
		} else {
		    backoffAt = &parsed
		}
```

Reply:
you're right. I didn't pay attention when setting the parsed value after an error.